coopaname_custom/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "coopaname_custom",
+    "summary": """
+        Lists all dependencies for Coopaname's deployement""",
+    "license": "AGPL-3",
+    "author": "Coop IT Easy SCRLfs",
+    "website": "www.coopiteasy.be",
+    "category": "Human Resources",
+    "version": "12.0.0.0.1",
+    "depends": ["contacts", "hr_contract", "hr_employee_firstname", "hr_recruitment"],
```

Comment:
Why are all these modules listed?

Reply:
We will list in the custom module all the modules we install at coopaname